### PR TITLE
[noiseprofiles] embiggen max length of maker array

### PR DESCRIPTION
**Pull request by johnny-bit**
this one sould help 7312 finally :) just wait till CI build completes.

### CODE REVIEW

**Comment by chhil** [comment]
I can build mine with...

image

**Comment by rawfiner** [comment]
I was puzzled, but @chhil found the solution! That's indeed the minItems and maxItems that needed to be changed! Thanks!

**Comment by johnny-bit** [comment]
yeah, changed in newest commit.

**Comment by johnny-bit** [comment]
yeah, this one works :)

**Comment by da-phil** [comment]
May I ask why we even need to enforce a min / max character length for camera brand name?
Those `minItems` / `maxItems` attributes could be also removed from the schema.

File: data/noiseprofiles.schema [hunk 1]
```diff
@@ -20,7 +20,7 @@
             "type": "string",
             "pattern": "^[a-zA-Z ]+$",
             "minLength": 3,
-            "maxLength": 13,
+            "maxLength": 14,
```
Comment (da-phil) [inline data/noiseprofiles.schema:23]: Could we get rid of those limits, as we already enforce to use one of the valid enum values further down?

**Comment by aurelienpierre** [comment]
Thanks !

**Comment by da-phil** [comment]
> May I ask why we even need to enforce a min / max character length for camera brand names?
> Those `minItems` / `maxItems` attributes could be also removed from the schema, because the schema already enforces the usage of only one of the listed enum values.

@johnny-bit / @rawfiner / @chhil  what do you think about that? I feel like this issue is not solved yet, we're over-complicating the process of adding new camera makers to the enum by imposing this additional length constraint, which we have to adapt each and every time we add a new camera maker name to the enum which exceeds the current limit.

**Comment by chhil** [comment]
I am relatively new to understanding the coding requirements. I can only speculate where sizes are needed to correctly  get displayed in the gui or some code generation , I don't know if there are historical reasons for it.

**Comment by johnny-bit** [comment]
@da-phil & @chhil  - there is no hard limit in code for getting gui nor do I see anything in code that would need such strict schema.

**Comment by da-phil** [comment]
> @da-phil & @chhil - there is no hard limit in code for getting gui nor do I see anything in code that would need such strict schema.

Exactly what I thought too.

**Comment by da-phil** [comment]
I created a new PR with a proposal and for further discussions as this one is already closed.

**Pull request merged**